src/nft.ts

```diff
@@ -201,8 +201,8 @@ export default class Nft extends FactoryBase {
       const { userAddress, ainftObjectId, name, symbol, slug, limit, cursor, order } = searchParams;
       query = { userAddress, ainftObjectId, name, symbol, slug, cursor, limit, order };
     }
-    const trailingUrl = `native/search/ainftObjects`;
-    return this.sendRequest(HttpMethod.GET, trailingUrl, query);
+    const trailingUrl = 'native/search/ainftObjects';
+    return this.sendRequestWithoutSign(HttpMethod.GET, trailingUrl, query);
```

Comment:
Could you leave some tsdoc comments about this?

Reply:
It's good idea! I'll make next pull request reflected this comment!

Reply:
please check https://github.com/ainft-team/ainft-js/pull/311 PR.